ProjectSettings/InputManager.asset

```diff
@@ -13,11 +13,11 @@ InputManager:
     positiveButton: right
     altNegativeButton: a
     altPositiveButton: d
-    gravity: 6
+    gravity: 3
```

Comment:
You will probably want to put this gravity change onto your own horizontal input system because this value might be used by someone else